content/doc/book/blueocean/creating-pipelines.adoc

```diff
@@ -0,0 +1,117 @@
+or from new `Jenkinsfile`s created with the
+<<pipeline-editor, Blue Ocean Pipeline Editor>>.
+The Pipeline Creation workflow guides users through this process in clear,
+easy to understand steps.
+
+== Starting Pipeline Creation
+
+At the top of the <<dashboard, Blue Ocean Dashboard>>,
+is a "*New Pipeline*" button that launch the Pipeline Creation workflow.
+
+image:blueocean/creating-pipelines/new-pipeline-button.png['Blue Ocean New Pipeline Button']
+
+On new a Jenkins instance, where there are no jobs or Pipelines
+and the Dashboard is empty, Blue Ocean will also display a message box offering
+to "Create a New Pipeline".
+
+image:blueocean/intro/new-pipeline-box.png['Blue Ocean Create Pipeline Message Box']
+
+== Creating a Pipeline for a Git Repository
+
+To create a Pipeline from a Git repository,
+start by selecting "Git" as the Source Control system.
+
+image:blueocean/creating-pipelines/select-git.png['Creating Pipeline from a Git Repository']
+
+Then enter the URL for the Git Repository and optionally select which credentials
+to use.  If the dropdown down does not show the desired credentials,
+they can be added using the "Add" button.
+
+When done, click "Create Pipeline".
+Blue ocean will look at all branches for the selected repository,
+and will start a Pipeline run for each branch containing a `Jenkinsfile`.
+
+image:blueocean/creating-pipelines/git-create-pipeline.png['Providing Git Repository Details']
+
+== Creating Pipelines for GitHub Repositories
+
+To create a Pipeline from a GitHub,
+start by selecting "GitHub" as the Source Control system.
+
+image:blueocean/creating-pipelines/select-github.png['Creating Pipeline for Github Repositories']
+
+=== Provide a GitHub Access Token
+
+If this is the first time Pipeline Creation has been run,
+Blue Ocean will ask for a
+link:https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/[GitHub Access Token]
+to allow Blue Ocean to access your organizations and repositories.
+
+image:blueocean/creating-pipelines/github-create-token.png['Add a Ocean GitHub Access Token']
+
+If you have not already created a access token, click on the link provided and
+Blue Ocean will navigate to the right page on GitHub,
+automatically selecting the appropriate permissions it will need.
+
+image:blueocean/creating-pipelines/github-personal-access-token.png['Creating a Blue Ocean GitHub Access Token']
+
+=== Select a GitHub Account or Organization
+
+All repositories on Github are grouped by owner, either an account or organization.
+When creating Pipelines, Blue Ocean mirrors that structure,
+asking users to select an account or organization which owns the repositories
+from which it will add Pipelines.
+
+image:blueocean/creating-pipelines/github-org.png['Selecting an Account or Organization']
+
+=== Single Repository or All Repositories
+
+Blue Ocean offers two styles of Pipeline creation, either "single Pipeline" or
+"discover all Pipelines".
```

Comment:
I think it would be helpful to make "single Pipeline" and "discover all Pipelines" interdocument cross-references to the corresponding sections of the document.